.golangci.yml

```diff
@@ -0,0 +1,28 @@
+run:
+  deadline: 300s
+  tests: true
+  skip-dirs:
+  - .git
+  - .tool
+  - vendor
+  - verify
+
+linters-settings:
+  dupl:
+    threshold: 100
+  gocyclo:
+    min-complexity: 50
+
+linters:
+  enable:
+  - govet
```

Comment:
The only one of the linters you enabled that does not fail in this repo is govet

Reply:
Are you sure? This set seems to pass (at least locally).

I just copied the file that is in repo-infra, don't really care about these values atm, just trying to get things working.

Reply:
Yes, if it passes, the bazel target doesn't work: Checking out your PR and running
```
golangci-lint run ./... 2>&1|c
```

gives  these https://gist.github.com/alvaroaleman/6ef68adef5aaa01633467c8eb8120772

Reply:
Hmm, what is golangci-lint's exit code? 

The implementation doesn't check output, just relies on the exit code to determine pass/fail: https://github.com/kubernetes/repo-infra/blob/ee974c891e8162941a25135e6c90575fbd35c93a/hack/verify-golangci-lint.sh#L49

Reply:
Hmm, exit code is 1

Reply:
```
$ golangci-lint run ./... &>/dev/null; echo $?
1
```

Reply:
Good news! It failed

Reply:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/21751/pull-test-infra-bazel/1380673894922326016